app/components/icons/bed.tsx

```
@@ -0,0 +1,17 @@
+const Bed: React.FC<React.SVGAttributes<{}>> = (props) => (
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)